### PR TITLE
Fix install command in README

### PR DESCRIPTION
**Pull request by leeavital**
#### What this PR does / why we need it:

The invocation of `helm install` in the README is incorrect.

### CODE REVIEW

File: charts/datadog/README.md [hunk 1]
```diff
@@ -42,7 +42,7 @@ By default, the Datadog Agent runs in a DaemonSet. It can alternatively run insi
 To install the chart with the release name `<RELEASE_NAME>`, retrieve your Datadog API key from your [Agent Installation Instructions](https://app.datadoghq.com/account/settings#agent/kubernetes) and run:
 
 ```bash
-helm install --name <RELEASE_NAME> \
+helm install <RELEASE_NAME> \
```
Comment (clamoriniere) [inline charts/datadog/README.md:45]: it depends if `helm2` or `helm3` is used, cf our public doc: https://docs.datadoghq.com/agent/kubernetes/?tab=helm

Comment (clamoriniere) [reply charts/datadog/README.md:45]: @leeavital 
could you update your PR according to my comment?
thanks

Comment (leeavital) [reply charts/datadog/README.md]: Sorry about that, I completely lost track of this PR. I've addressed the comments.

**Comment by clamoriniere** [comment]
@leeavital 
I think you made a mistake when you have updated your PR.
Could you rebase your PR again and check why the result is empty?
thanks

**Pull request closed without merging**